Example Apps/Example Swift/ProxyManager.swift

```diff
@@ -49,13 +49,14 @@ extension ProxyManager {
     }
 
     /// Attempts to close the connection between the this app and the car's head unit. The `SDLManagerDelegate`'s `managerDidDisconnect()` is called when connection is actually closed.
-    func resetConnection() {
+    func stopConnection() {
         guard sdlManager != nil else {
-            delegate?.didChangeProxyState(ProxyState.stopped)
+            delegate?.didChangeProxyState(.stopped)
             return
         }
 
         sdlManager.stop()
+        delegate?.didChangeProxyState(.stopped)
```

Comment:
I would move `delegate?.didChangeProxyState(.stopped)` to the `managerDidDisconnect` method.